sklearn/feature_selection/tests/test_variance_threshold.py

```diff
@@ -32,17 +33,16 @@ def test_variance_threshold():
         assert (len(data), 1) == X.shape
 
 
+@pytest.mark.skipif(np.var(data2) == 0,
```

Comment:
```suggestion
@pytest.mark.skipif(np.var([[-0.13725701]] * 10) == 0,
```
no need to define data.

Reply:
Then I'd need to redefine data inside the function.